cfgov/cfgov/settings/production.py

```diff
@@ -144,3 +144,12 @@
     LOGGING["loggers"]["mozilla_django_oidc"] = {
         "level": "INFO",
     }
+
+if DEPLOY_ENVIRONMENT == "container":
+    MIDDLEWARE += (
+        "whitenoise.middleware.WhiteNoiseMiddleware",
+    )
+
+    # This can result in a redirect loop if True and an X-Forwarded-Proto
+    # header is not provided.
+    SECURE_SSL_REDIRECT = False
```

Comment:
More generally, how would you feel about creating a separate settings file for (legacy) EC2 configuration for things in the baseline that we don't need to complicate our container-based deploys for now? So perhaps there's `settings.ec2` that inherits from `settings.production`? This wouldn't necessarily need to be part of this particular PR.

Reply:
I think that would make sense. It makes me laugh a little bit because I went back and forth between just doing that and using `DEPLOY_ENVIRONMENT` like I ended up committing.

Generally, I think would like to clean up our settings, and I think this suggestion might make more sense as part of that?